.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -4,34 +4,43 @@ about: Fill out this template before submitting a Pull Request.
 title: Pull Request
 labels: 'pull request'
 assignees: ''
-
 ---
+<!--- Provide a general summary of your changes in the Title field above -->
+<!--- Please select appropriate Priority, Status,and Type labels -->
+<!--- If you do not have permission to select labels please state which labels you would like -->
 
-<!--- 
-Title: Provide a summarize changes in the pull request title above. 
--->
-
-<!---
-Tags: please select one or more relevant tags
---->
+<!--- please select one or more relevant tags --->
```

Comment:
Because GitHub doesn't use "tags" in its user interface, I'd drop this comment. The one about "labels" above seems sufficient & unambiguous.

Reply:
Thanks for you feedback! I agree and have made the change

Reply:
You're welcome :-)